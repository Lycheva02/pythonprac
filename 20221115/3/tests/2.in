alp = Alpha()
alp.i = 13
alq = AlphaQ()
alq.u = 14
print(alp, alq, sep = ' ; ')