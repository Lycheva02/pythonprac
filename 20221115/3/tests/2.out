i: 13 ; u: 14
